### PR TITLE
add expander on ellipsis

### PR DESCRIPTION
**Pull request by JefMari**
For issue #141 
I base the syntax in [Primer css utilities](http://primercss.io/utilities/)

image

### CODE REVIEW

File: styles/_utils.scss [hunk 1]
```diff
@@ -69,6 +69,10 @@
 
 ._ellipsis {
   @include text-ellipsis;
+
+  &.-expander:hover {
+    max-width: 10000px !important;
```
Comment (srph) [inline styles/_utils.scss:74]: Is this how primer implements the expander?

File: docs/sections/Docs-Utilities/index.jsx [hunk 2]
```diff
@@ -76,11 +75,15 @@ export default class Utilities extends React.Component {
...
         <section className="doc-bottom-space-large">
-          <h1 className="doc-heading">Text Ellipsis</h1 >
+          <h1 className="doc-heading">Text Ellipsis</h1>
           <p className="lead">Helpful for collapsing very long text.</p>
           <p className="_text-muted">As a class, use the <code>._ellipsis</code> class; as a mixin, use <code>text-ellipsis</code>. As in <code>@include text-ellipsis;</code>.</p>
           <div className="doc-ellipsis-example-container">
-            <p className="_ellipsis">This is a very long text, aaand aaand.. yolo please swag.</p>
+            <p className="_ellipsis">This is a very long text, aand aaand.. yolo please swag.</p>
+          </div>
+          <p className="_text-muted _spacer">Adding <code>-expander</code>, you can reveal the entire text on hover</p>
```
Comment (srph) [inline docs/sections/Docs-Utilities/index.jsx:84]: `.-expander`, note the `.`.

File: styles/_utils.scss (hunk 1 above)
Comment (JefMari) [inline styles/_utils.scss:74]: yep

---

image

---

any suggestion?

File: docs/sections/Docs-Utilities/index.jsx (hunk 2 above)
Comment (JefMari) [inline docs/sections/Docs-Utilities/index.jsx:84]: woops got it

File: styles/_utils.scss (hunk 1 above)
Comment (srph) [inline styles/_utils.scss:74]: OK, no problem.

**Comment by srph** [comment]
Thanks.

**Pull request merged**